internal/service/fw/model_atcfw_pop_region.go

```diff
@@ -0,0 +1,97 @@
+package fw
+
+import (
+	"context"
+
+	"github.com/hashicorp/terraform-plugin-framework/attr"
+	"github.com/hashicorp/terraform-plugin-framework/diag"
+	schema "github.com/hashicorp/terraform-plugin-framework/resource/schema"
+	"github.com/hashicorp/terraform-plugin-framework/types"
+	"github.com/hashicorp/terraform-plugin-framework/types/basetypes"
+
+	"github.com/infobloxopen/bloxone-go-client/fw"
+
+	"github.com/infobloxopen/terraform-provider-bloxone/internal/flex"
+)
+
+type AtcfwPoPRegionModel struct {
+	Addresses types.List   `tfsdk:"addresses"`
+	Id        types.Int64  `tfsdk:"id"`
+	Location  types.String `tfsdk:"location"`
+	Region    types.String `tfsdk:"region"`
+}
+
+var AtcfwPoPRegionAttrTypes = map[string]attr.Type{
+	"addresses": types.ListType{ElemType: types.StringType},
+	"id":        types.Int64Type,
+	"location":  types.StringType,
+	"region":    types.StringType,
+}
+
+var AtcfwPoPRegionResourceSchemaAttributes = map[string]schema.Attribute{
+	"addresses": schema.ListAttribute{
+		ElementType:         types.StringType,
+		Optional:            true,
+		MarkdownDescription: "PoP Region's IP addresses",
+	},
+	"id": schema.Int64Attribute{
+		Computed:            true,
+		MarkdownDescription: "The PoP Region's serial, unique ID",
+	},
+	"location": schema.StringAttribute{
+		Optional:            true,
+		MarkdownDescription: "PoP Region's location",
+	},
+	"region": schema.StringAttribute{
+		Optional:            true,
+		MarkdownDescription: "PoP Region's name",
+	},
+}
+
+func ExpandAtcfwPoPRegion(ctx context.Context, o types.Object, diags *diag.Diagnostics) *fw.AtcfwPoPRegion {
```

Comment:
since PoP regions don't have any resources, the expand function might be unused. You can delete it if unused.

Reply:
Deleted